testing/integration-tests/src/full_client/client/mod.rs

```diff
@@ -242,10 +242,10 @@ async fn unsigned_extrinsic_is_same_shape_as_polkadotjs() {
     // - start local substrate node.
     // - open polkadot.js UI in browser and point at local node.
     // - open dev console (may need to refresh page now) and find the WS connection.
-    // - create a balances.transfer to ALICE with 12345 and "submit unsigned".
+    // - create a balances.transferAllowDeath to ALICE (doesn't matter who from) with 12345 and "submit unsigned".
     // - find the submitAndWatchExtrinsic call in the WS connection to get these bytes:
     let expected_tx_bytes = hex::decode(
-        "b004060700d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d0f0090c04bb6db2b"
+        "b004060000d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d0f0090c04bb6db2b"
```

Comment:
Question: could we also arrive at these bytes constructing a call via subxt-signer and the static interface?

Reply:
We wouldn't even need to sign anything :)

This call is really just checking that, for a given random unsigned extrinsic, subxt (via the static interface above) produces the same bytes that somehting like Polkadot.js produces. ie we want to compare the Subxt bytes with some non-subxt bytes and if they align, we gain some confidence that subxt constructed them properly :)

Reply:
Aah I see, yup we could have done that too :) But at least by checking the whole hash against a different client, we get a bit of extra reassurance that _everything_ subxt does is in line at least with the other client; any part we do using Subxt code itself will no longer be "compared"